server/container/filesystem-api/src/test/java/org/apache/james/filesystem/api/FileSystemFixture.java

```diff
@@ -17,22 +17,23 @@
  * under the License.                                           *
  ****************************************************************/
 
-package org.apache.james.utils;
+package org.apache.james.filesystem.api;
 
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.InputStream;
+import java.net.URL;
+import java.util.Optional;
 
-import org.apache.commons.lang.NotImplementedException;
-import org.apache.james.filesystem.api.FileSystem;
+import org.apache.commons.lang3.NotImplementedException;
 
 public class FileSystemFixture {
 
     public static final FileSystem THROWING_FILE_SYSTEM = new FileSystem() {
         @Override
         public InputStream getResource(String url) throws IOException {
-            throw new NotImplementedException();
+            throw new NotImplementedException("Not implemented");
```

Comment:
What is the added value of this message?
The class is enough in the stacktrace, IMO.

Reply:
This message is compulsary with commans-lang-3. Before it uses commons-lang. Hence I added it even if the value is low.